integration_tests/blockstack_integration_tests/scenarios/stacks_patch_genesis_block_files_big.py

```diff
@@ -0,0 +1,119 @@
+genesis_patches_files = dict([(str(int(k) - 613250 + 691), v) for (k, v) in blockstack.lib.genesis_block.GENESIS_BLOCK_PATCHES_FILES.items()])
+
+consensus = "17ac43c1d8549c3181b200f1bf97eb7d"
+
+wallets = []
+
+def scenario( wallets, **kw ):
+    patch_addrs_path = blockstack.lib.config.get_genesis_bulk_address_path("blockchain-airdrop-2020.csv")
+    patch_file_contents = open(patch_addrs_path).read().strip()
+    patch_file_addrs = map(lambda a: str(a.strip()), filter(lambda a: len(a.strip()) > 0, patch_file_contents.split('\n')))
+    assert len(patch_file_addrs) == 322146
+
+    for a in patch_file_addrs:
+        try:
+            virtualchain.address_reencode(a)
+        except Exception as e:
+            print "not a valid address: {}".format(a)
+            traceback.print_exc();
+            raise e
+    
+    blockstack.lib.config.set_genesis_block_patches_files(genesis_patches_files)
+    testlib.set_account_audits(False)
+
+    print json.dumps(genesis_patches_files, indent=4, sort_keys=True)
+    
+    for i in range(0, len(patch_file_addrs)/10000):
+        assert genesis_patches_files.get(str(i + 691)) is not None
+        assert genesis_patches_files[str(i + 691)]['metadata'] == 'batch-{}'.format(i+1)
+
+    testlib.next_block(**kw)
+
+    times = []
+
+    for i in range(0, (len(patch_file_addrs)/10000)+1):
+        t1 = time.time()
+        testlib.next_block(**kw)    # make the balances real (end of 691 + i, start of 692 + i)
+        t2 = time.time()
+       
+        db_path = os.path.join(os.environ['BLOCKSTACK_WORKING_DIR'], 'blockstack-server.db')
+        db = namedb.namedb_open(db_path)
+        cur = db.cursor()
+        cur.execute('BEGIN')
+        for j in range(i*10000, min(len(patch_file_addrs), (i+1)*10000)):
+            addr = virtualchain.address_reencode(patch_file_addrs[j].strip())
+            acct = namedb.namedb_get_account(cur, addr, 'STACKS')
+            assert acct is not None, "no account {} (the {}'th one)".format(addr, j)
+            assert namedb.namedb_get_account_balance(acct) == 100 * 10**6
+        cur.execute('END')
+        
+        print "Pocessing time: {}".format(t2 - t1)
+        times.append(t2 - t1)
+        db.close()
+
+    print "Total processing times"
+    for t in times:
+        print "{}".format(t)
+       
+    db_path = os.path.join(os.environ['BLOCKSTACK_WORKING_DIR'], 'blockstack-server.db')
+    db = namedb.namedb_open(db_path)
+    cur = db.cursor()
+    cur.execute('BEGIN')
+    for i in range(0, len(patch_file_addrs)):
+        if i % 1000 == 0:
+            print "Tested {} addresses".format(i)
+
+        addr = virtualchain.address_reencode(patch_file_addrs[i].strip())
+        acct = namedb.namedb_get_account(cur, addr, 'STACKS')
+        assert acct is not None, "no account {} (the {}'th one)".format(addr, i)
+        assert namedb.namedb_get_account_balance(acct) == 100 * 10**6
+    cur.execute('END')
+
+def check( state_engine ):
+    return True
```

Comment:
Are we supposed to do something with state_engine before returning `True`?

Reply:
For this test, there's nothing needed, since all the test logic occurs in the prior method.